shell.nix

```diff
@@ -30,14 +30,15 @@ let
 in mkShell {
   buildInputs = [
     bash
+    cpio
     curl
     docker
     git
     git-lfs
     gnumake
     gnused
     libarchive
-    minio
+    minio-client
```

Comment:
Is this even used anymore?

Reply:
actually yes, we use it to upload to s3. We could use a different tool, but there isn't really a need to change it in this PR.

Reply:
ahh, ok.